portfolio/src/main/webapp/script.js

```diff
@@ -95,3 +87,23 @@ function deleteComments() {
     const request = new Request('/delete-data', {method: 'POST'});
     fetch(request).then(response => response.json()).then(refreshComments(0));
 }
+
+/** Creates an <p> element containing text. */
+function createCommentElement(comment) {
+  const divElement = document.createElement('div');
+  const nameElement = document.createElement('p');
+  const pElement = document.createElement('p');
+  const imgElement = document.createElement('img');
+  const brElement = document.createElement('br');
+
+  nameElement.innerText = comment[1].name;
+  pElement.innerText = comment[1].text;
+  imgElement.setAttribute('src', comment[1].image);
```

Comment:
You shouldn't need the [1] if you treat comments as an array as my comment above.